### PR TITLE
Try to re-enable RequiresCapability tests

### PR DESCRIPTION
**Pull request by agocke**
Found some non-determinism in finding the right file -- it should now look through all the files.

### CODE REVIEW

File: test/ILLink.RoslynAnalyzer.Tests/TestCaseUtils.cs [hunk 1]
```diff
@@ -14,40 +14,33 @@
 
 namespace ILLink.RoslynAnalyzer.Tests
 {
-	public class TestCaseUtils
+	public abstract class TestCaseUtils
```
Comment (tlakollo) [inline test/ILLink.RoslynAnalyzer.Tests/TestCaseUtils.cs:17]: Why is this now an abstract class?

Comment (agocke) [reply test/ILLink.RoslynAnalyzer.Tests/TestCaseUtils.cs:17]: It only contains helpers -- it probably should have been an abstract class in the beginning.

In fact, everything in it is static. It should be a static class, but XUnit requires that all `MemberData` work against members in the same class, so the only way to reuse the functionality is to put it in a base class and use inheritance.

File: test/ILLink.RoslynAnalyzer.Tests/LinkerTestCases.cs [hunk 2]
```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections.Generic;
+using System.Linq;
```
Comment (tlakollo) [inline test/ILLink.RoslynAnalyzer.Tests/LinkerTestCases.cs:6]: Nit: additional using

**Review by tlakollo — APPROVED** [review]
Looks good to me

**Pull request merged**